mentorship ios/Settings/Settings.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  Settings.swift
+//  Created on 20/06/20
+//  Created for AnitaB.org Mentorship-iOS 
+//
+
+import SwiftUI
+
+struct Settings: View {
+    var settingsModel = SettingsModel()
+    //Alert used for logout and delete action, to confirm user's action before proceeding.
+    @State var showAlert = false
+    @State var alertTitle = ""
+    
+    var body: some View {
+        NavigationView {
+            Form {
+                //Top Space
+                Section { EmptyView() }
+                
+                //Section 1 of settings : about, feedback, and change password
+                Section {
+                    ForEach(0 ..< settingsModel.settingsOptions[0].count, id: \.self) { index in
+                        NavigationLink(destination: Text("Section 1 setting")) {
+                            //Setting list cell
+                            HStack {
+                                //icon image
+                                Image(systemName: self.settingsModel.settingsIcons[0][index])
+                                    .padding(.trailing, DesignConstants.Padding.insetListCellFrameExpansion)
+                                    .font(.system(size: DesignConstants.Fonts.Size.insetListIcon))
+                                
+                                //text
+                                Text(self.settingsModel.settingsOptions[0][index])
+                            }
+                        }
+                    }
+                }
+                
+                //Section 2 of settings : logout, and delete account
+                Section {
+                    ForEach(0 ..< settingsModel.settingsOptions[1].count, id: \.self) { index in
+                        //setting list cell
+                        HStack {
+                            //icon image
+                            Image(systemName: self.settingsModel.settingsIcons[1][index])
+                                .padding(.trailing, DesignConstants.Padding.insetListCellFrameExpansion)
+                            
+                            //button
+                            Button(self.settingsModel.settingsOptions[1][index]) {
+                                self.alertTitle = self.settingsModel.settingsOptions[1][index]
+                                self.showAlert.toggle()
+                            }
+                            .foregroundColor(.red)
+                        }
+                    }
+                }
+            }
+            .navigationBarTitle("Settings")
+            .environment(\.horizontalSizeClass, .regular)
+            .alert(isPresented: $showAlert) {
+                Alert(
```

Comment:
Refactoring alert and making function which returns alertView would be great, as It will help us to reuse it when needed

Reply:
I don't see how we can refactor the alert.

.alert used to control the presentation is already a modifier, and Alert in itself has various different initializers for different use cases.

If you have any specific implementation in mind, please let me know.

Reply:
Not referring to .alert, You are right Alert can work for many use cases. Unlike UIKit alert, these are pretty good😅.